roadmap.qmd

```diff
@@ -1,158 +1,215 @@
-| {{< var status.wip >}} | Ongoing | [`seedcase-project/team`](https://github.com/seedcase-project/team) | Documentation specific to the Seedcase team, like onboarding, common configuration files, and meeting agendas and minutes. |
-| {{< var status.wip >}} | Ongoing | [`seedcase-project/community`](https://github.com/seedcase-project/community) | Content for community building, outreach, and contributing guidelines for the Seedcase Project |
-| {{< var status.wip >}} | Ongoing | [`seedcase-project/seedcase-website`](https://github.com/seedcase-project/seedcase-website) | Main website for the Seedcase Project |
-| {{< var status.wip >}} | Early-to-mid 2025 | [`sprout.seedcase-project.org`](https://sprout.seedcase-project.org/docs/guide) usage guide | Guide for using and interacting with `seedcase-sprout`. |
-| {{< var status.wip >}} | Late 2025 | Flower usage guide | Guide for using and interacting with `seedcase-flower` |
-| {{< var status.wip >}} | Mid 2026 | Propagate usage guide | Guide for using and interacting with `seedcase-propagate` |
-| {{< var status.wip >}} | Late 2026 | Garden usage guide | Guide for using and interacting with `seedcase-garden` |
-
-: List of documentation deliverables with approximate timelines for
-uploading to an archive for the first time.
-
-### Training and education
-
-| Status | Timeline | Deliverable | Description |
-|------------------|------------------|-------------------|-------------------|
-| {{< var status.planned >}} | TBD | Tutorial | Installing and setting up Seedcase on a server so Data Resources can be created |
-| {{< var status.planned >}} | TBD | Tutorial | Using Seedcase to create and manage a Data Resource |
-| {{< var status.planned >}} | TBD | Tutorial | Requesting to start a data project for a Data Resource |
-
-: List of training and education deliverables with approximate timelines
+#### Tutorials
+
+Tutorials are learning-oriented documentation that includes multi-day
+courses, online tutorials, and learning shorts. These will be developed
+as [open educational
+resources](https://en.wikipedia.org/wiki/Open_educational_resources).
+
+| Status | Learning resource |
+|-----------------------------------------------|-------------------------|
+| {{< var status.wip >}} | [`rostools/git4cats`](https://github.com/rostools/git4cats): :cat: Git for Collaborative Teams: An introductory course on using Git and GitHub for team-based collaborative workflows and practices |
+| {{< var status.planned >}} | [`rostools/r-pkg-intro`](https://github.com/rostools/r-pkg-intro): Build research software using R as a solo developer in research environments. |
+| {{< var status.potential >}} | Course: Iterative project management and development for effective collaboration and work. |
+| {{< var status.potential >}} | Course: Introduction to building research software using Python as a solo developer in research environments. |
```

Comment:
Change order. So, `r-pkg-intro` swaps with `python-pkg-intro` and change `r-pkg-intro` status to potentiel